pybaseball/statcast_pitcher.py

```diff
@@ -25,3 +29,78 @@ def statcast_pitcher(start_dt: Optional[str] = None, end_dt: Optional[str] = Non
     df = split_request(start_dt, end_dt, player_id, url)
 
     return df
+
+@cache.df_cache()
+def statcast_pitcher_exitvelo_barrels(year: int, minBBE: Union[int, str] = "q") -> pd.DataFrame:
+    url = f"https://baseballsavant.mlb.com/leaderboard/statcast?type=pitcher&year={year}&position=&team=&min={minBBE}&csv=true"
+    res = requests.get(url, timeout=None).content
```

Comment:
This isn't a problem with this PR specifically, but we're doing the same thing over and over here and in statcast_batter. I know I made `get_statcast_data_from_csv_url` in `datasources.statcast` but that uses the `try_parse_dataframe` that seems to slow things down quite a bit. I would suggest that we make a new function there like `get_statcast_data_from_csv_url_raw` that does all of this repetitive work for us without the overhead of `try_parse_dataframe`.

Reply:
I would suggest a shorter name if you decide to go that route

Reply:
Can we just use `requests.get(url, kwargs)` for the other parameters?

Reply:
Only if you removed them from the original url and named them exactly as they appear in the query strings I believe.

Reply:
I did mess around with using a `base_url = f"https://baseballsavant.mlb.com/leaderboard/{leaderboard_name}"` variable and then passing a `params = {"type": "pitcher", "minBBE": "q"...}` dict to the requests call, ie `requests.get(base_url, params=params)`. This would probably be more flexible and would maybe allow us to basically bring all of the statcast functions into one file where the names aren't so long, but IDK if I really want to embark on a refactor that could break people's code. 

I'll check out the stuff in `datasources.statcast`